packages/dd-trace/test/runtime_metrics.spec.js

```diff
@@ -521,7 +521,8 @@ function createGarbage (count = 50) {
               }
               if (metric === 'runtime.node.cpu.total') {
                 assert(
-                  number >= expected && number <= expected + 1,
+                  // Lower numbers can happen due to rounding issues.
```

Comment:
It's not clear from the comment what you're doing to fix it. I think this should help:

```suggestion
                  // Subtracting 0.02 since lower numbers can happen due to rounding issues.
```